addons/account/models/account_move.py

```diff
@@ -2353,6 +2353,8 @@ def write(self, vals):
             for move in self:
                 if 'tax_totals' in vals:
                     super(AccountMove, move).write({'tax_totals': vals['tax_totals']})
+                if 'journal_id' in vals:
+                    move.line_ids._check_constrains_account_id_journal_id()
```

Comment:
- We should make it on self as we don't need to loop to call that method
- We should put an empty line above to separate it from the previous block as it's not linked to the quick edit stuff
```suggestion

            if 'journal_id' in vals:
                self.line_ids._check_constrains_account_id_journal_id()
```